nibabel/streamlines/array_sequence.py

```diff
@@ -37,7 +37,7 @@ def __init__(self, arr_seq, common_shape, dtype):
         self.common_shape = common_shape
         n_in_row = reduce(mul, common_shape, 1)
         bytes_per_row = n_in_row * dtype.itemsize
-        self.rows_per_buf = bytes_per_row / self.bytes_per_buf
+        self.rows_per_buf = self.bytes_per_buf / bytes_per_row
```

Comment:
Should `rows_per_buf` be an `int` or a `float`? If the idea is to load whole rows at a time, then you may want to use `//` to ensure that you round down.